index.html

```diff
@@ -193,6 +193,16 @@
             ['table', 'telemetry.plot.overlay', 'telemetry.plot.stacked'],
             {indicator: true}
         ));
+        openmct.install(openmct.plugins.SynchronizedFolder({
```

Comment:
I love the idea of having a generic plugin that adds a "virtual" folder to the tree that includes Couch search results, this is genius. See my general comments on how I think we can tweak this using Object Providers and Composition Providers rather than object interceptors.

A couple of thoughts on the naming, it only works with Couch DB, so the name should reflect that, and we only support synchronization of plans right now, so we should remove `Synchronized` from the name.

Reply:
Done.